adapters/pubmatic/usersync.go

```diff
@@ -0,0 +1,15 @@
+package pubmatic
+
+import (
+	"net/url"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+func NewPubmaticSyncer(cfg *config.Configuration) usersync.Usersyncer {
```

Comment:
Another way to do this method is to have a signature like:
```
func NewPubmaticSyncer(externalURL string, adapter config.Adapter) usersync.Usersyncer
```
which would avoid the need for some adapters to do a lookup on the adapter map.

Reply:
Either way works. I do agree with your comment here... reducing the work adapters have to do is definitely an advantage. I can't think of any disadvantages, either, so I'll give a small edge to `NewPubmaticSyncer(externalURL string, adapter config.Adapter)`.

It's probably not a huge deal, though.

Reply:
I think the main reason I kept it that way was in case there were only configuration additions. I say we keep it for now and change it later if it makes sense.